ArkEcosystem/laravel-foundation
# refactor: destroy tippy only for outdated elements

## Summary

https://app.clickup.com/t/1t7hjx5

The issue described on the ticket is because the current implementation removes all the tippy instances (and creates them again) every time livewire updates the DOM.

The was intended to fix an issue with tippy tooltips that never disappeared when an element disappears from the DOM while have the cursor over keeping the tooltip forever.

This PR refactors the implementation to check for every previous tippy instance and remove the ones that are no longer on the DOM

### To test

- Merge the changes from this PR on nodem
- The issue described on the ticket should not happen anymore
- Also you need ways to emulate an element disappearing on the DOM while the cursor is over an element or find places when you can see that issue. I personally hardcoded conditions to randomly hide a tooltip and then tested manually.
- This PR affects every place where we have tippy implementations so you may need to think in potential problems with my new implementation

## Checklist
